docs/9_EgenskaperModellkatalog.adoc

```diff
@@ -0,0 +1,179 @@
+=== Modellkatalog [[modellkatalog]]
+
+Klassen _Modellkatalog_ er obligatorisk.
+
+==== Obligatoriske egenskaper for klassen _Modellatalog_
+
+===== Modellkatalog: beskrivelse [[modellkatalog-beskrivelse]]
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| description
+|URI| dct:description
+|Range| rdfs:Literal
+|Beskrivelse| Fritekst-beskrivelse av innholdet i katalogen. Egenskapen bør gjentas for parallelle språkversjoner av beskrivelsen.
+|Multiplisitet| 1..n
+|Status| Obligatorisk
+|===
+
+===== Modellkatalog: identifikator [[modellkatalog-identifikator]]
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| identifier
+|URI| dct:identifier
+|Range| rdfs:Literal
+|Beskrivelse| Hovedidentifikator for katalogen, for eksempel en URI eller annen identifikator som er stabil og globalt unik.
+|Multiplisitet| 1..1
+|Status| Obligatorisk
+|===
+
+===== Modellkatalog: modell
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| model
+|URI| modelldcatno:model
+|Range| modelldcatno:InformationModel
+|Beskrivelse| Kobler katalogen til informasjonsmodeller som er en del av katalogen.
+|Multiplisitet| 1..n
+|Status| Obligatorisk
+|===
+
+===== Modellkatalog: tittel [[modellkatalog-tittel]]
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| title
+|URI| dct:title
+|Range| rdfs:Literal
+|Beskrivelse| Inneholder navnet på katalogen. Egenskapen kan bli gjentatt for parallelle språkversjoner av navnet.
+|Multiplisitet| 1..n
+|Status| Obligatorisk
+|===
+
+===== Modellkatalog: utgiver [[modellkatalog-utgiver]]
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| publisher
+|URI| dct:publisher
+|Range| foaf:Agent
+|Beskrivelse| Refererer til en aktør (organisasjon) som er ansvarlig for å gjøre katalogen tilgjengelig. Bør være autoritativ URI for enhet, sekundært organisasjonsnummer.
```

Comment:
"... URI for enhet" --> "... URI for aktøren". 
(Jeg oppdaterer også DCAT-AP-NO)